modules/main-rules/pages/transport.adoc

```diff
@@ -1,39 +1,61 @@
-= Transport
+= Transportation
 
-On some unit data sheets you'll see that the unit has the {transport} special ability.
-We explain special abilities in more detail at xref:special-rules:transport-x.adoc[], but transport is a vital aspect of the Movement phase so we'll discuss it here too.
+On some unit data sheets you'll see that the unit has one of two special abilities that relate to transportation: {transport} or {hitch}.
+We explain these and other special abilities in more detail elsewhere; but transport is a vital aspect of the Movement phase so we'll discuss it here too.
 
-Units that have the {transport} special ability can carry other units across the battlefield.
+When we speak of these transportation abilities, we use the terms 'embark/embarked' and 'disembark/disembarked' to describe actions and states that relate to the units that you transport.
 
-* To embark or disembark other units (or cargo) it costs the {transport} unit 5cm of its movement.
-* An embarking or disembarking unit must also pay 5cm of its movement.
-* You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
-* You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example. 
+Both the {transport} and {hitch} abilities indicate units that can carry other units across the battlefield, but they work in slightly different ways.
+
+== Transportation by units that have the {transport} ability
+
+Units that have the {transport} ability can carry other units across the battlefield.
+They can only transport Infantry units -- except where the rules or army lists say otherwise.
+
+To embark or disembark a unit:
+
+* You must sacrifice 5 cm from the movement of the {transport} unit.
+* You must sacrifice 5 cm from the movement of the unit that you embark or disembark.
+* You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement allowance.
```

Comment:
What's a 'movement allowance'?

Reply:
Pretty much a cop-out until we've finished honing the whole 'move' thing. Potentially 'movement' is more than a single 'move', so your 'movement allowance' is the total of all the moves you're doing right now, if it is more than one move (ie, March orders).

Feel free to suggest an alternative that works.

Reply:
> Feel free to suggest an alternative that works.

'Movement'? 😜

More seriously, it's what the rest of the page is using for that concept, which is why 'movement allowance' stood out so much.

Reply:
But it's a very minor issue.

Reply:
Yeah, I might've had some nuance in mind that set me on a tangent, but I can't think of it now. Maybe a brain fart. Movement will do. :-)
```suggestion
* You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
```